Front-End/challenge1/script.js

```diff
@@ -1 +1,19 @@
+/* eslint-disable linebreak-style */
 // Modify this file only
+const counter = document.getElementById('counter');
+let count = parseInt(document.getElementById('counter').innerHTML, 10);
+const increaseBtn = document.getElementById('increase');
+const decreaseBtn = document.getElementById('decrease');
+
+let result = 0;
+const more = 1;
+const less = -1;
+
+function counterHandler(modify) {
+	count += modify;
+	result = count;
+	counter.innerHTML = result;
```

Comment:
Why don't just do something like `const counterHandler = () => counter.innerHTML = modify`

Reply:
Adding to this, this might feel a bit like a nitpick but I feel naming like `modifier` would be more appropriate than `modify`. Verbs sound better when applied to functions and nouns to variables.
Besides that, in this specific scenario having single responsibility functions such as a single `increment` and `decrement` function might be more readable.